.travis.yml

```diff
@@ -7,7 +7,7 @@ dist: xenial
 cache: pip
 
 before_install:
-    - pip install docutils
+    - pip install docutils pygments
```

Comment:
I don't think we can just add `pygments` to get the CI working, because presumably the actual deployment code does not include `pygments`, no?

At the very least, I think that this should be a separate change, not done as part of adding this PEP.

Reply:
Is it okay if CI fails or should I also remove the code highlighting?

Reply:
We can't merge the PR with the CI failing, so I would remove it unless you want to block this PR on the side-quest of getting pygments support in the PEPs infrastructure. Not sure how feasible that is (and in fact maybe it already exists and we really _do_ just need to add it to Travis).

Reply:
Removed! 🙂

Reply:
Fine to try it in another PR, but looks like all thar2really is needed is to install pygments:

1. The `script` step calls `make` to turn plaintext PEPs into HTML, and I assume pygments is only needed here to add colour to the generated HTML files (I'm afk this week so cannot verify this assumption.now).

2. Then the `deploy` step calls `deploy.bash` which does `make package` to `tar.gz` the plaintext and generated files and images, and then calls `aws` to actually deploy the `tar.gz` to AWS.

Reply:
Good to know. I think we should do it in a separate PR, though, because though I feel comfortable merging "Add a PEP", I don't feel comfortable merging something that changes the CI/CD pipeline.

Reply:
https://github.com/python/peps/pull/1577